cmd/juju/ssh/ssh_jump.go

```diff
@@ -45,6 +55,7 @@ type sshJump struct {
 	container              string
 	target                 string
 	args                   []string
+	knownHostsPath         string
```

Comment:
s/knownHostsPath/tempKnownHostsPath. 

And can we please describe & be kind to ourselves what this sshJumpProvider does on the struct via a godoc